modules/themes/bloodyHellMate.js

```diff
@@ -0,0 +1,27 @@
+const version = "0";
+
+let style;
+
+let obj = {
+  onImport: async () => {
+    goosemodScope.showToast("onImport()");
+    style = document.createElement("style");
+    document.head.appendChild(style);
+    style.appendChild(
+      document.createTextNode(
+        `@import url("https://raw.githack.com/MrDragonBoi/bloody-hell-mate/main/index.css")`
+      )
+    );
+  },
+
+  remove: async () => {
+    style.remove();
+  },
+
+  name: "Bloody Hell Mate",
+  description: "Makes everything bloody red, mate!",
+  author: ["MrDragonBoi", "Liam The Protogen"],
```

Comment:
Still unsure whether we should add porter as an author.

Reply:
This is true... I was kinda in a rush to make this, sorry

Reply:
No worries. More of a reminder / call to self that we should think about this in the future.